### PR TITLE
Cliqz and Ghostery as system addons

### PR DESCRIPTION
**Pull request by spacifici**

### CODE REVIEW

File: mozilla-release/mobile/android/base/Makefile.in [hunk 1]
```diff
@@ -40,46 +26,29 @@ chrome-%::
...
-
-.PHONY: update-generated-wrappers update-fennec-wrappers
-
 # This target is only used by IDE integrations. It rebuilds resources
 # that end up in omni.ja using the equivalent of |mach build faster|,
 # does most of the packaging step, and then updates omni.ja in
 # place. If you're not using an IDE, you should be using |mach build
 # mobile/android && mach package|.
-$(ABS_DIST)/$(MOZ_APP_NAME)/$(OMNIJAR_NAME): FORCE
+$(ABS_DIST)/fennec/$(OMNIJAR_NAME): FORCE
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/Makefile.in]: This should still be `$(MOZ_APP_NAME)`

File: mozilla-release/mobile/android/base/Makefile.in [hunk 2]
```diff
@@ -90,7 +59,7 @@ gradle-targets: $(generated_resources) $(generated_files)
 
 # Local developers update omni.ja during their builds.  There's a
 # chicken-and-egg problem here.
-gradle-omnijar: $(abspath $(DIST)/$(MOZ_APP_NAME)/$(OMNIJAR_NAME))
+gradle-omnijar: $(abspath $(DIST)/fennec/$(OMNIJAR_NAME))
```
Comment (spacifici) [inline mozilla-release/mobile/android/base/Makefile.in]: Also this

File: mozilla-release/Makefile.in [hunk 3]
```diff
@@ -199,13 +199,10 @@ binaries::
 	@$(MAKE) install-manifests install_manifests=dist/include
 endif
 
-# Cliqz Start
```
Comment (sharath-cliqz) [inline mozilla-release/Makefile.in]: We still need the `Cliqz Start` comment, as we are downloading the Cliqz Artifacts.

**Comment by spacifici** [comment]
Merged

**Comment by luciancor** [comment]
@spacifici how is this working on updates?

**Pull request closed without merging**